README.md

```diff
@@ -402,8 +402,8 @@ arXiv, 2021.
 *[Standard and Event Cameras Fusion for Feature Tracking](https://dl.acm.org/doi/10.1145/3459066.3459075)*,  
 Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://github.com/LarryDong/FusionTracking)
 - <a name="Mondal21iccvw"></a>Mondal, A., Shashant, R., Giraldo, J. H., Bouwmans, T., Chowdhury, A. S.,  
-*[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://www.researchgate.net/publication/354462913_Moving_Object_Detection_for_Event-based_Vision_using_Graph_Spectral_Clustering)*,  
-IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021.
+*[Moving Object Detection for Event-based Vision using Graph Spectral Clustering](https://openaccess.thecvf.com/content/ICCV2021W/GSP-CV/html/Mondal_Moving_Object_Detection_for_Event-Based_Vision_Using_Graph_Spectral_Clustering_ICCVW_2021_paper.html)*,  
```

Comment:
Hi. If the html page has the same information as the pdf (i.e., there is no supplementary material), I suggest providing just one link here (to the pdf, like the link below).

Reply:
Yes, you can provide the link to the PDF only.

Reply:
could you commit the change and then we merge?